root/static/scripts/common/i18n/history.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * @flow strict
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as wrapGettext from './wrapGettext.js';
+
+export const l_history: (string) => string =
+  wrapGettext.dgettext('history');
+
+export const ln_history: (string, string, number) => string =
+  wrapGettext.dngettext('history');
+
+export const lp_history: (string, string) => string =
+  wrapGettext.dpgettext('history');
```

Comment:
Minor: The two latter are unused, so probably unneeded.

(Same as in a previous comment)